docs/resources/resources-glossary.md

```diff
@@ -109,15 +105,17 @@ See also: [tutorial](#tutorial).
 
 A **ribbon** is a command bar that organizes an application's features into a series of tabs or buttons at the top of a window. A **ribbon button** is one of the buttons within this series. See [Show or hide the ribbon in Office](https://support.microsoft.com/office/d946b26e-0c8c-402d-a0f7-c6efa296b527#ID0EBBD=Newer_Versions) for more information.
 
-## runtime
+## Runtime
```

Comment:
```suggestion
## runtime
```
I think Alison and Alex decided that the glossary entries are lowercase, except when a proper noun.